### PR TITLE
Live 1182 remove editions embeds

### PR DESCRIPTION
**Pull request by joecowton1**
## Why are you doing this?

Removing atoms and embeds as part of Editions embed fixes.

### Changes
- remove Editions audio atom
- remove Editions callout
- remove Editions quiz atom
- remove Editions Spotify embed
- remove Editions chart atom

### CODE REVIEW

**Comment by abeddow91** [comment]
`http://localhost:8080/music/2021/apr/07/baker-boy-hiatus-kaiyote-montaigne-and-others-australias-best-new-music-for-april?editions` still shows a Spotify embed

**Comment by joecowton1** [comment]
> `http://localhost:8080/music/2021/apr/07/baker-boy-hiatus-kaiyote-montaigne-and-others-australias-best-new-music-for-april?editions` still shows a Spotify embed (assuming this is a spotify embed)

This is because although `EmbedKind.Spotify` is no longer rendering for Editions, not all Spotify Embeds are being correctly parsed and some are instead rendered as generic embeds.

**Comment by JamieB-gu** [comment]
> not all Spotify Embeds are being correctly parsed and some are instead rendered as generic embeds as in this case.

Just to clarify, there are two ways Spotify can end up in the page. One is as a formal "audio" embed, which we treat as a distinct element and which Joe has filtered out for Editions. The other is as a "generic" embed, which is a catch-all for all sorts of different embed types and is sometimes used (perhaps incorrectly) to include Spotify.

I wouldn't say these are being parsed incorrectly as such - it's a conscious decision to [only trigger the audio parsing code](https://github.com/guardian/apps-rendering/blob/f803ffa1797a0cf17d3b4b9df5f16f6a0822b639/src/bodyElement.ts#L405) when encountering an audio element. We _could_ decide to start looking for certain patterns in generic embeds and convert them to the more formal type. Thus far we haven't done this because that'll cause a difference in how they're rendered and thus might surprise the people who chose this method of embedding.

File: src/components/embed.tsx [hunk 1]
```diff
@@ -13,22 +13,23 @@ import type { FC } from 'react';
 
 interface Props {
 	embed: Embed;
+	editions?: boolean;
```
Comment (JamieB-gu) [inline src/components/embed.tsx:16]: Could we make this explicit?

```suggestion
	editions: boolean;
```

**Review by JamieB-gu — APPROVED** [review]
Nice, surprisingly few changes required!

One question.

File: src/components/embed.tsx (hunk 1 above)
Comment (joecowton1) [reply src/components/embed.tsx]: Good call, done.

**Review by JamieB-gu — APPROVED** [review]

**Pull request merged**